python/cog/schema.py

```diff
@@ -58,6 +58,7 @@ class PredictionResponse(PredictionBaseModel):
     output: t.Any
 
     id: t.Optional[str]
+    version: t.Optional[str]
```

Comment:
I *think* this is ok, as it shouldn't change anything about what we're sending back in webhooks (because of https://github.com/replicate/cog/blob/b4a940efd8a7790f96d5fcda7b50be68751ca875/python/cog/json.py#L28) but let's just check that.

Reply:
If the input prediction comes with the version (which it does in Replicate's case), then it will be sent in the webhook too. I'm not sure whether that was the case before or not, or if API just overrides it or not. But it is [in the schema](https://replicate.com/docs/reference/http#get-prediction), so I'm not sure I understand what the problem here might be, or how that code is relevant?

Reply:
It seems like `version` is the only field that wasn't typed out here. Might it be that we just aren't sending it in webhooks right now..? Wouldn't that be a bug that this is fixing?